webpack.config.js

```diff
@@ -8,7 +8,6 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      // title: 'Output Management',
```

Comment:
- [ ] [OPTIONAL] Please, consider removing comments to make your code clear 👍

Reply:
@Mo7amedSMahdi , thanks for the review. Let me work on the changes.